subsys/bluetooth/shell/bt.c

```diff
@@ -193,65 +193,128 @@ static bool data_cb(struct bt_data *data, void *user_data)
 	}
 }
 
-static void print_data_set(uint8_t type, uint8_t set_value_len,
+static void print_data_hex(const uint8_t *data, uint8_t len, bool warning)
```

Comment:
You could consider changing `bool warning` to `enum shell_vt100_color color` so the caller can call it with `SHELL_WARNING` or `SHELL_INFO`, but it is also OK as is